workflows/phylorthology.nf

```diff
@@ -48,66 +70,416 @@ include { INPUT_CHECK } from '../subworkflows/local/input_check'
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/fastqc/main'
-include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
-include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoftwareversions/main'
-
+include { BUSCO as BUSCO_SHALLOW                } from '../modules/nf-core/busco/main'
+include { BUSCO as BUSCO_BROAD                  } from '../modules/nf-core/busco/main'
+include { BUSCO                                 } from '../modules/nf-core/busco/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_TEST } from '../modules/nf-core/diamond/blastp/main'
+include { DIAMOND_BLASTP                        } from '../modules/nf-core/diamond/blastp/main'
+include { MAFFT                                 } from '../modules/nf-core/mafft/main'
+include { MAFFT as MAFFT_REMAINING              } from '../modules/nf-core/mafft/main'
+include { IQTREE                                } from '../modules/nf-core/iqtree/main'
+include { IQTREE as IQTREE_REMAINING            } from '../modules/nf-core/iqtree/main'
```

Comment:
A comparable place I've seen this is the nf-core genomeassembler workflow that is in progress:
https://github.com/nf-core/genomeassembler/blob/dev/workflows/genomeassembler.nf

They do this for using the same module on hifi, hic, ont, or illumina data:

```
include { BUILD_FASTK_DATABASES as BUILD_HIFI_FASTK_DATABASE     } from "$projectDir/subworkflows/local/build_fastk_database"
include { BUILD_FASTK_DATABASES as BUILD_HIC_FASTK_DATABASE      } from "$projectDir/subworkflows/local/build_fastk_database"
include { BUILD_FASTK_DATABASES as BUILD_ONT_FASTK_DATABASE      } from "$projectDir/subworkflows/local/build_fastk_database"
include { BUILD_FASTK_DATABASES as BUILD_ILLUMINA_FASTK_DATABASE } from "$projectDir/subworkflows/local/build_fastk_database"
```

Reply:
My assumption is this is a different way to achieve this instead of each data type is a subworkflow, for example having all illumina processing being a subworkflow that is only called with an `--illumina` parameter.

Reply:
If you don't name twice it won't let you run it without renaming. Since I'm running some of these modules repeatedly with either different parameters or different datasets, I need to either use renamed modules, or write new ones (often not necessary. 

But, I can get rid of the third, orignally named BUSCO module since I don't actually call it.

Reply:
But yeah, as to your point about using a subworkflow for this - down the line it would probably be good to implement it this way as it would tidy things up significantly I think! Potentially can wait to make those changes though if it'll be a significant lift.